CHANGELOG.md

```diff
@@ -29,10 +29,14 @@ You can check your current version with the following command:
     ```
 
 For more information, see [UP42 Python package description](https://pypi.org/project/up42-py/).
+## 2.1.1a4
+
+**Oct 29, 2024**
+- Updating endpoint for `base::get_credits_balance` and adding `timestamp` as an optional parameter.
```

Comment:
We can't add new functionality with patch version, let's drop the timestamp part